src/Http/IsRequest.php

```diff
@@ -68,9 +68,9 @@ public function getQuery(): array
         return $this->query;
     }
 
-    public function session(): SessionManager
+    public function session(): Session
     {
-        return get(SessionManager::class);
+        return get(Session::class);
```

Comment:
Why don't we inject this?

Reply:
I'm gonna keep it as is for now, I think. The alternative is that we make session nullable, which isn't the most user friendly, because we can assume that if you request the session, you actually need it (and it needs to be started if it doesn't exist)